eclipse/launchers/GameRunner.launch

```diff
@@ -8,6 +8,7 @@
 <listEntry value="1"/>
 </listAttribute>
 <booleanAttribute key="org.eclipse.debug.ui.ATTR_LAUNCH_IN_BACKGROUND" value="false"/>
+<stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.buildship.core.classpathprovider"/>
```

Comment:
This attribute was apparently added with Oxygen.3. This is the first time this launch configuration has been edited since the release of Oxygen.3. The other two launch configurations have been recently edited, and this attribute is already present in those files.